src/fluid/braginskii/bragViscosity.cpp

```diff
@@ -0,0 +1,931 @@
+        }
+        bi /= Bmag;
+        bj /= Bmag;
+        bk /= Bmag;
+
+        #if GEOMETRY == CARTESIAN
+          bbgradV = EXPAND( bi*bi*dVxi + bj*bi*dVxj + bk*bi*dVxk,
+                            + bi*bj*dVyi + bj*bj*dVyj + bk*bj*dVyk,
+                            + bi*bk*dVzi + bj*bk*dVzj + bk*bk*dVzk);
+
+          divV = D_EXPAND(dVxi, + dVyj, + dVzk);
+
+          Pnor_par = 3.0*etaBrag*(bbgradV - divV/3.0);
+
+          tau_xy = Pnor_par*bi*bj;
+          tau_yy = Pnor_par*(bj*bj - 1./3.);
+          tau_yz = Pnor_par*bj*bk;
+        // No source term in cartesian geometry
+        #else
+          [[maybe_unused]] real vx1i, vx2i, vx3i;
+          vx1i = vx2i = vx3i = ZERO_F;
+
+          EXPAND(  vx1i = 0.5*(Vc(VX1,k,j-1,i)+Vc(VX1,k,j,i)); ,
+                   vx2i = 0.5*(Vc(VX2,k,j-1,i)+Vc(VX2,k,j,i)); ,
+                   vx3i = 0.5*(Vc(VX3,k,j-1,i)+Vc(VX3,k,j,i)); )
+
+          Pnor_parC = 3.0*etaBragC*(bbgradVC - divVC/3.0);
+        #endif // GEOMETRY != CARTESIAN
+
+        #if GEOMETRY == CYLINDRICAL
+          bbgradV = D_EXPAND(
+                       bi*bi*dVxi + bi*bj*dVyi + bi*bk*(dVzi  - vx3i/x1(i)),
+                     + bj*bi*dVxj + bj*bj*dVyj + bj*bk*dVzj + bk*bk*vx1i/x1(i),
+                       );
+
+          divV = D_EXPAND(dVxi + vx1i/x1(i) ,
+                            + dVyj ,
+                             );
+          // No cylindrical geometry in 3D!
+          // No source term along the z-axis in cylindrical/polar coordinates
+          EXPAND( bragViscSrc(IDIR,k,j,i) = ZERO_F;  ,
+                  bragViscSrc(JDIR,k,j,i) = ZERO_F;  ,
+                  bragViscSrc(KDIR,k,j,i) = ZERO_F;  )
+        #elif GEOMETRY == POLAR
+          bbgradV = EXPAND(
+                      bi*bi*dVxi                         + bj*bi*1./x1(i)*dVxj
+                                                                                  + bk*bi*dVxk,
+                    + bi*bj*(dVyi - vx2i/x1(i)) + bj*bj*(1./x1(i)*dVyj + vx1i/x1(i))
+                                                                                  + bk*bj*dVyk,
+                    + bi*bk*dVzi                         + bj*bk*1./x1(i)*dVzj
+                                                                                  + bk*bk*dVzk
+                    );
+
+          divV = D_EXPAND(dVxi + vx1i/x1(i) ,
+                           + 1./x1(i)*dVyj ,
+                           + dVzk);
+
+          // See Geoffroy's trick for curvature source terms in cylindrical/polar coordinates
+          EXPAND( bragViscSrc(IDIR,k,j,i) = ZERO_F;  ,
+                  bragViscSrc(JDIR,k,j,i) = ZERO_F;  ,
+                  bragViscSrc(KDIR,k,j,i) = ZERO_F;  )
+//          //cell-centered values for source terms
+//          tau_xyC = Pnor_parC*biC*bjC;
+//
+//          EXPAND( bragViscSrc(IDIR,k,j,i) = ZERO_F;  ,
+//                  bragViscSrc(JDIR,k,j,i) = tau_xyC/x1(i);  ,
+//                  bragViscSrc(KDIR,k,j,i) = ZERO_F;  )
```

Comment:
I'd rather we avoid merging commented code without a comment; the way I see it, either we'll never need it and it should be removed, or it might be useful in the future and we need a comment that explains _why_ we're not throwing it, otherwise it'll just stay there for a while, and probably even after the reason to keep it around has faded away.

Reply:
I agree and I will remove those in the next commit, I'll just leave the comment 

```diff
@@ -651,16 +651,10 @@
+          // See Geoffroy's trick for curvature source terms in curvilinear coordinates
```

to warn other users that some the curvilinear "source" terms arising from the divergence of a tensor in curvilinear coordinates are handled with special tricks by Geoffroy.